README.md

```diff
@@ -28,16 +35,51 @@ This section describes the terms used for all documentation in this repository.
   create an environment to host the container. Neither the guest OS nor the
   guest kernel need to be the same as the host operating system.
 
-  See [the image builder documentation](image-builder/README.md).
+  See the [image creation](#image-creation) section.
 
 - initrd (or "initramfs")
 
-  A compressed cpio archive loaded into memory and used as part of the Linux
-  startup process. During startup, the kernel unpacks it into a special
-  instance of a tmpfs that becomes the initial root file system.
+  A compressed cpio archive, created from a rootfs which is loaded into memory and used as part of the Linux startup process. During startup, the kernel unpacks it into a special instance of a tmpfs that becomes the initial root filesystem.
```

Comment:
Suggested rewrite: A compressed cpio archive created from a rootfs, which is loaded into memory and used as part of the Linux startup process. During startup, the kernel unpacks it into a special instance of a tmpfs that is the initial root filesystem.